core-library/src/main/java/org/silverpeas/core/admin/domain/driver/ldapdriver/LDAPUser.java

```diff
@@ -22,6 +22,7 @@
 
 import com.novell.ldap.LDAPConnection;
 import com.novell.ldap.LDAPEntry;
+import org.silverpeas.core.SilverpeasExceptionMessages;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.silverpeas.core.SilverpeasExceptionMessages'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)